### DISCUSSION TITLE
ImageMagick can't VideoFileClip

### DISCUSSION THREAD

**Discussion by mryt66** [Zulko/moviepy#2087] [Q&A] [answered]
OSError: MoviePy Error: creation of None failed because of the following error:

[WinError 2] Nie można odnaleźć określonego pliku. (windows can't find specified file)

.This error can be due to the fact that ImageMagick is not installed on your computer, or (for Windows users) that you didn't specify the path to the ImageMagick binary in file conf.py, or that the path you specified is incorrect

**Comment by SohamTilekar** [comment, marked as answer]
Download the ImageMagika and Put it in the Path Variable 
or
Provide the path to the magick.exe

**Reply by mryt66** [reply]
It didn't help I have it in my Path variables, but still gives this same error message
![image](https://github.com/Zulko/moviepy/assets/64143856/5ef7cae0-0134-4046-95cc-224d18a05c82)

**Reply by mryt66** [reply]
It did work somehow
